src/control/differential/DifferentialControl.hpp

```diff
@@ -32,7 +32,7 @@ class DifferentialControl : public Control
     void setSpeed(int speed) override;
 
     /* Check `Control` interface for documentation */
-    void overrideMotorSpeed(int firstMotorSpeed, int secondMotorSpeed) override;
+    virtual void overrideMotorSpeed(int firstMotorSpeed, int secondMotorSpeed) override;
```

Comment:
:warning: **modernize-use-override** :warning:
'virtual' is redundant since the function is already declared 'override'
```suggestion
    virtual void ```